stacks/apps/dovetail-analytics.yml

```diff
@@ -347,6 +347,276 @@ Resources:
         - { Key: prx:dev:application, Value: Analytics }
 
   # DynamoDB
+  AnalyticsDynamoDbTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: id
+          AttributeType: S
+      BillingMode: PROVISIONED
+      KeySchema:
+        - AttributeName: id
+          KeyType: HASH
+      ProvisionedThroughput:
+        ReadCapacityUnits: !If [IsProduction, 100, 1]
+        WriteCapacityUnits: !If [IsProduction, 400, 1]
```

Comment:
I'm not entirely sure how we got the RCU/WCU numbers for prod?  It's currently scaled all the way down to 1 on read capacity.  And all the way down to 215 (the minimum we specified) for write capacity.  Maybe trying to handle traffic spikes?